features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/root/PreferencesRootPresenter.kt

```diff
@@ -82,7 +77,7 @@ class PreferencesRootPresenter @Inject constructor(
             logoutState = logoutState,
             myUser = matrixUser.value,
             version = versionFormatter.get(),
-            showCompleteVerification = sessionIsNotVerified,
+            showCompleteVerification = canVerifySession,
```

Comment:
About the naming, using `can` does not enforce the fact that the user `should` verify the session. IIUC, when true, the user `should` and `can` verify the session. Not sure, there is a modal verb to reflect this, so maybe just name the boolean `showCompleteVerification`?